change/@fluentui-react-image-94f41cca-af00-4429-981c-3b258ef16bf4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
Should we bump the version? It should affect DOM structure

Reply:
Yes you're right, it does affect DOM. I'll change this to `patch` instead. Thanks